### ISSUE TITLE
"Copy Code to Clipboard" button copies non-breaking spaces

### ISSUE THREAD

**Issue by austb** [notable/notable#778]
- **OS Version**: macOS Mojave 10.14.6
- **Notable Version**: 1.7.0

### Current behavior

When copying a command with the copy to clipboard button (such as the one below). The space characters are copied as non-breaking spaces (U+00A0 : NO-BREAK SPACE [NBSP]). This breaks when pasting to the command line because the terminal (well bash's parser to be specific) doesn't split fields on non-breaking spaces.

I verified this with the help of http://babelstone.co.uk/Unicode/whatisit.html.

When you highlight and copy with cmd-c, the regular space characters are copied and it works as expected.

Example Notable code field

<pre lang="no-highlight"><code>
```bash
bundle exec
```
</code></pre>

The error message from the command line 
```
$ bundle exec
-bash: bundle exec: command not found
```

### Expected behavior

Copying with the Copy Code to Clipboard button copies regular space characters (U+0020 : SPACE [SP])

**Comment by austb** [comment]
Verified this is still occuring on the most recent version of Notable, 1.7.2.

I opened a pull request with a potential solution.

**Comment by fabiospampinato** [comment]
Good catch, I'll fix this in v1.7.3.

**Comment by fabiospampinato** [comment]
I just fixed this, the fix will be available in the next release.
